contracts/UFragments.sol

```diff
@@ -51,19 +49,8 @@ contract UFragments is ERC20Detailed, Ownable {
         _;
     }
 
-    // Precautionary emergency controls.
-    bool public rebasePaused;
-    bool public tokenPaused;
-
-    modifier whenRebaseNotPaused() {
-        require(!rebasePaused);
-        _;
-    }
-
-    modifier whenTokenNotPaused() {
-        require(!tokenPaused);
-        _;
-    }
+    bool public rebasePausedDeprecated;
```

Comment:
As far I recall making this private should not affect anything. It will also remove these variable from the ABI.

But If we want to be extra-cautious, I'm okay with renaming.  👍 

Leaving it to you to decide

Reply:
From the docs, only order and type determine storage layout: https://solidity.readthedocs.io/en/v0.4.24/miscellaneous.html#layout-of-state-variables-in-storage

Also for most recent v0.7.0: https://solidity.readthedocs.io/en/v0.7.0/internals/layout_in_storage.html?highlight=storage%20layout

This is corroborated by OpenZeppelin's high level guide: https://docs.openzeppelin.com/upgrades/2.6/writing-upgradeable

So I'll move to `private`, good idea.

Reply:
If you want additional certainty on stability of storage layout it could be a good idea to run integration tests on forked mainnet state

Reply:
Good ideas thanks!

We have a script that dumps storage state so we can compare before and after.

We also have a mainnet-staging that we test with before mainnet production.